docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md

```diff
@@ -0,0 +1,131 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update credentials for a vCenter Server data source credentials in VMware Aria Operations for Networks.
+
+
+## Syntax
+
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>] [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+```
+
+## Description
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update credentials for a vCenter Server data source in VMware Aria Operations for Networks.
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.
+
+## Examples
+
+### Example 1
+
+Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
+
+This example updates a configured vCenter Server with updated credentials as supplied in VMware Aria Operations for Networks.
+
+```
+
+This example updates a configured vCenter Server with updated values as supplied in VMware Aria Operations for Networks.

```

Comment:
```suggestion
### Example 1

Update-AriaNetworksvCenterDataSourceCredentials -id 15832:902:2623605245375371420 -CollectorId 15832:901:1711011916294613031 -fqdn sfo-m01-vc01.sfo.rainpole.io -username svc-inv-vsphere -password VMw@re1! -nickname "sfo-m01-vc01 - Management Domain vCenter Server"
```

This example updates the credentials for vCenter Server data source in VMware Aria Operations for Networks.

```